strongbox-rest-client/src/test/java/org/carlspring/strongbox/rest/client/RestAssuredArtifactClient.java

```diff
@@ -366,4 +366,58 @@ public String search(String repositoryId,
                            .asString();
     }
 
+    public Headers getHeadersfromHEAD(String url)
```

Comment:
I think this should be `getHeadersFromHEAD`

Reply:
+1